### ISSUE TITLE
[Bug] Parser breaks on declares by ordinal

### ISSUE THREAD

**Issue by sancarn** [SSlinky/VBA-LanguageServer#19]
**Describe the bug**

Parser breaks on declares by ordinal:

```vb
Private Declare PtrSafe Function IUnknown_GetWindow Lib "shlwapi" Alias "#172" (ByVal pIUnk As IUnknown, ByVal hwnd As LongPtr) As Long
```

Should be a simple fix, currently antlr grammar is as follows:

```antlr
aliasClause: ALIAS wsc STRINGLITERAL;
```

Needs changing to

```antlr
aliasClause: ALIAS wsc stringOrOrdinal;
stringOrOrdinal
     : STRINGLITERAL
    | ORDINALSTRING
    ;
ORDINALSTRING
    : '"#' [0-9]+ '"'
    ;
```

I can likely PR, but just not sure how to build/test... Any instructions? 😅

**Comment by SSlinky** [comment]
Have been a bit MIA sorry! Too much work on. I'd love it if you were able to invest some time.

I've written a [contributing](https://github.com/SSlinky/VBA-LanguageServer/blob/master/contributing.md) page if that helps. Appreciate that it's not super straight forward getting started with this project.

**Comment by sancarn** [comment]
> Have been a bit MIA sorry! Too much work on. I'd love it if you were able to invest some time.

I feel ya 👍 Thanks, for the contributing page, will see if I can get it up and running on my end to enable PRs😄

**Comment by sancarn** [comment]
> I've written a [contributing](https://github.com/SSlinky/VBA-LanguageServer/blob/master/contributing.md) page if that helps. 

@SSlinky What do you mean by "Use the Client + Server launch configuration"? I can't see any scripts in the client nor server... Is there any guidance on this online?

**Comment by SSlinky** [comment]
Ahh, this is me assuming you're using vscode to develop. I was referring to this file - [.vscode/launch.json](../blob/master/.vscode/launch.json). Should be able to see it in the list of launch configurations.

![image](https://github.com/user-attachments/assets/1398fb4a-560e-4ab8-ab0e-bafea8c56a69)

**Comment by sancarn** [comment]
Ahhh that makes a lot of sense 😅 Maybe worth adding to contributing markdown 😁 I'll give that a go ☺️

**Comment by SSlinky** [comment]
How did you go, mate? Let me know if you want me to jump in and take credit for your work. Also, I've just committed to both master and dev so you'll need to pull :)

**Comment by sancarn** [comment]
Hey @SSlinky, I apologise I haven't had an awful lot of time recently to get back in the swing of TS 😅 by all means take the credit I'm sure there will be more I can contribute on 😁

**Comment by SSlinky** [comment]
Somewhat of a red herring but I got there eventually. It was an issue with enum being declared `LongPtr` as this is not an ambiguous name. All good. I've added this specific type of enum declaration and it's all working again. Published patch 1.4.5.
